components/admin_console/blockable_link/index.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators, Dispatch} from 'redux';
+import {GenericAction} from 'mattermost-redux/types/actions';
+
+import {deferNavigation} from 'actions/admin_actions';
+import {getNavigationBlocked} from 'selectors/views/admin';
+
+import BlockableLink from './blockable_link';
+
+function mapStateToProps(state: object) {
+    return {
+        blocked: getNavigationBlocked(state)
+    };
+}
+
+function mapDispatchToProps(dispatch: Dispatch<GenericAction>) {
+    return {
+        actions: bindActionCreators(
```

Comment:
Same, here, I would keep the old code styling, because is more consistent with the rest of our code base.

Reply:
All sorted. My linter got a bit carried away.